solc/CommandLineInterface.cpp

```diff
@@ -69,6 +69,16 @@ namespace dev
 namespace solidity
 {
 
+extern bool g_hasOutputed;
+std::ostream& sout(bool used = true) {
+	g_hasOutputed = g_hasOutputed || used;
+	return cout;
+}
+std::ostream& serr(bool used = true) {
+	g_hasOutputed = g_hasOutputed || used;
+	return cerr;
+}
+
```

Comment:
As a safeguard, we could even add `#define cout ERROR` or something like that here.

Reply:
sure, I will define it to empty to disable its usage.